src/data-workspace/inputs/generic-input.js

```diff
@@ -43,23 +43,41 @@ export const GenericInput = ({
         )
     }
 
+    const formatValue = (value) => {
+        let formattedValue = value
+
+        if (
+            value &&
+            value.trim() &&
+            NUMBER_TYPES.includes(valueType) &&
+            Number.isFinite(Number(value))
+        ) {
+            formattedValue = Number(value).toString()
+        }
+
+        return formattedValue
+    }
```

Comment:
No need to call `.trim()` on line 54 because:
```js
Number(' .0001  ').toString()
// result: '0.0001'
Number(' 0000.0001  ').toString()
// result: '0.0001'
```